src/main/java/io/strimzi/kafka/metrics/KafkaMetricsCollector.java

```diff
@@ -87,16 +87,14 @@ public MetricSnapshots collect() {
         for (Map.Entry<MetricName, KafkaMetric> entry : metrics.entrySet()) {
             MetricName metricName = entry.getKey();
             KafkaMetric kafkaMetric = entry.getValue();
-            LOG.trace("Collecting Kafka metric {}", metricName);
 
             String prometheusMetricName = metricName(metricName);
-            // TODO Filtering should take labels into account
```

Comment:
As discussed in https://github.com/strimzi/metrics-reporter/issues/5, we decided to not implement that feature now. So removing the `TODO`. Same in `YammerMetricsCollector` below.